main/plugins/open-with-plugin.js

```diff
@@ -1,17 +1,18 @@
 'use strict';
 const path = require('path');
 const {getAppsThatOpenExtension, openFileWithApp} = require('mac-open-with');
+const {getFormatExtension} = require('../common/constants');
 
 const action = async context => {
   const filePath = await context.filePath();
   openFileWithApp(filePath, context.appUrl);
 };
 
 const apps = new Map(
-  ['mp4', 'gif', 'apng', 'webm']
-    .map(extension => [
-      extension,
-      getAppsThatOpenExtension.sync(extension)
+  ['mp4', 'gif', 'apng', 'webm', 'av1']
```

Comment:
This pretty much adds the same apps as `mp4` since they both have an `.mp4` extension (and that's how we determine which apps open what file).

Some apps don't support it, but there's no way to know. Gifski has a pretty good error message, VLC supports it, and even Kap does (it opens in the preview just fine)